forge/src/multi_runner.rs

```diff
@@ -272,7 +285,7 @@ mod tests {
 
     fn test_multi_runner() {
         let mut runner = runner();
-        let results = runner.test(&Filter::new(".*", ".*")).unwrap();
+        let results = runner.test(&Filter::with_all_pass()).unwrap();
```

Comment:
regarding naming,
perhaps `matches_all` would be more appropriate here?

Reply:
Yeah, that one bothered me - `matches_all` is much better